data/greenlight/dotenv

```diff
@@ -1,30 +1,32 @@
-# POSTGRES DATABASE URL
+### POSTGRES DATABASE URL
 # Must be in the format postgres://username:password@host:port
 #   E.g. postgres://postgres:password@postgres:5432
-DATABASE_URL=
-
-# REDIS CACHE URL
+DATABASE_URL=postgres://postgres:password@postgres:5432
```

Comment:
The intention of this is the same as for `SECRET_KEY=`, to force the person deploying to set those values properly.

Reply:
Users in the docs then  should be requested to set the **DATABASE_URL** manually which if left defaulted will be **postgres://postgres:password@postgres:5432**.
So we will pointing users to do that manually or through running commands.
Because we're trying to keep the steps as simple and straightforward as possible so that users can spin GL up and running easily I set those defaults while mentioning the requirement to change them unless for testing purposes.